REQUIRE

```diff
@@ -2,7 +2,6 @@ julia 0.6
 Compat 0.24
 FixedPointNumbers 0.3.0
 ColorTypes 0.2.7
-Images 0.6
+ImageCore
 FileIO
-@osx Homebrew
-BinDeps
+BinaryProvider
```

Comment:
Given the comment  at the top of `deps/build.jl` this needs a minimum version number

Reply:
good catch!